src/App.js

```diff
@@ -1,18 +1,23 @@
-import React, { Component } from 'react';
-import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/pages/Navbar';
+import Home from './components/pages/Home';
+import Quotes from './components/pages/Quotes';
+import './index.css';
 
-export default class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+const App = () => (
+  <>
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route path="/math-magicians-app" element={<Home />} />
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quote" element={<Quotes />} />
+      </Routes>
+    </Router>
+  </>
+);
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214764677-f93ce0a2-4f4a-4830-9baa-1a4a68fc29a5.png)

**[Optional]**

- [ ] Some of your commit messages are not meaningful enough :fearful: Such as here all your commit messages are about updating the readme file but what are you updating in your readme I am not able to understand :thinking: Kindly make your commit messages meaningful to make your pull request professional and it will also help you in tracking the commits. So, please follow this requirement in your upcoming milestones as well :+1: